### DISCUSSION TITLE
Website for designing flag of Iran /سایت برای طراحی پرچم ایران

### DISCUSSION THREAD

**Discussion by parhamsms** [WeAreMahsaAmini/Core#364] [Ideas]
the new society of Iran needs a new flag. The symbol of the lion and the sun (شیروخورشید) comes from the imperial dictatorship and the word Allah comes from the Islamic dictatorship.
In my opinion, the logo in the middle of the flag should be suitable for today's democratic society.

I suggest that a website be designed so that people can present their ideas for the Iran flag. 

My suggestion is the lotus flower in the middle of the flag. A symbol of peace, femininity, prosperity in hardship, and ancient civilization

![Gol-mikhi](https://user-images.githubusercontent.com/108494211/193350143-4c30366b-4228-4efc-aa18-cd75ef2c9ebb.png)
![2560px-Flag_of_Iran_(1964–1980) svg](https://user-images.githubusercontent.com/108494211/193350187-1b98d018-a11d-41f7-9f1e-8a635d147d6b.png)

**Comment by Ariannnnnn** [comment]
I like the idea of lotus, what do you think about having a native flower? I did a quick Google search and Water Lily is the national flower of Iran. Maybe something like this: https://stock.adobe.com/images/lotus-flower-abstract-logo-water-lily-silhouette/189370775

**Reply by freedomfighterforiran** [reply]
Lotus was used in Persepolis city during Achaemenian Empire. See the references below:

https://fa.wikipedia.org/wiki/%D9%86%DB%8C%D9%84%D9%88%D9%81%D8%B1_%D8%A2%D8%A8%DB%8C_%D8%AF%D8%B1_%D9%81%D8%B1%D9%87%D9%86%DA%AF_%D8%A7%DB%8C%D8%B1%D8%A7%D9%86_%D8%A8%D8%A7%D8%B3%D8%AA%D8%A7%D9%86

https://www.researchgate.net/figure/Depiction-of-lotus-5-th-century-BCE-Persepolis-Iran_fig3_350378862

**Comment by kaveh6** [comment]
I think the flag is something that all Iranian people need should agree on . we could have a platform to invite designers to send their designs and available people to vote for the one they like .

**Reply by Ariannnnnn** [reply]
100% agree

**Reply by erfan7emz** [reply]
It's a great idea and we definitely need a website like this to vote on the flag and other things that everyone should agree on. I would love to work/collaborate on this project.

**Comment by parhamsms** [comment]
I suggest that a website be designed so that people can present their ideas for the Iran flag.

**Comment by sorsaffari** [comment]
You must be unaware. I'm not against a new flag, but the flag you are referring to is simply the former flag of Iran/Persia and it has been around centuries before our grandparents were born.

Also, we should do what we can to help this revolution succeed. Not just anything!

**Reply by kaveh6** [reply]
I'm not against the old flag . I think we should all respect our old flag and be united under that flag to be able to win this . 
but on the other hand , I also think that Iranian renaissance is something a lot more important than our last 1400 years which our culture was tainted with Islamic believes . a new Era need a new flag . I agree we should respect our culture and embrace it , but at the same time I'd like to have a new flag as a symbol of a new beginning .

**Comment by amastaneh** [comment]
I suggest the Shiro Khorshid as ancient symbol and the symbol of "Iranian unity" should be chosen.

![image](https://user-images.githubusercontent.com/6195199/193669790-71e938f1-33b8-42f0-a4ca-f28c907f8bfd.png)
![image](https://user-images.githubusercontent.com/6195199/193678432-51d3499f-287f-4235-9f6d-7a3acd9a9bfa.png)

**Reply by kaveh6** [reply]
Right now , the only legit flag that we have before Islamic Republic is Shiro Khorshid flag and we should respect it and be proud to stand under it and be united with the rest of Iranian . 
and having a new Iran flag is not something that should be discussed right now .
